cmake/modules/Findtim-vx.cmake

```diff
@@ -18,6 +18,8 @@
 #    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #    DEALINGS IN THE SOFTWARE
 #
+set(TIM_VX_ENABLE_40BIT      ON)
```

Comment:
Can we only enable this only if multi_device enabled?
And 40BIT should not always enabled, multi_device is a standalone feature with 40bit.